doc/changelog/11-standard-library/16914-deprecate-ZModulo.rst

```diff
@@ -0,0 +1,6 @@
+- **Deprecated:** :g:`Cyclic.ZModulo.ZModulo` because there have been no 
```

Comment:
```suggestion
- **Deprecated:** :g:`Cyclic.ZModulo.ZModulo` because there have been no
```

Reply:
Linter fails from extra trailing space

Reply:
@JasonGross Note that it would be OK for you to commit this suggestion before merging. That being said, if @andres-erbsen does it, this could be an opportunity to squash the commits that amend each other.